redshift_connector/cursor.py

```diff
@@ -239,9 +244,31 @@ def executemany(self: "Cursor", operation, param_sets) -> "Cursor":
         self._row_count = -1 if -1 in rowcounts else sum(rowcounts)
         return self
 
+    def insert_data_bulk(self, filename, url, name, links, url_index, name_index):
```

Comment:
Can type annotations be added to the method parameters?

Reply:
This method appears to only support inserting values to 2 columns. Is there anything preventing us from supporting a variable number of columns?

Reply:
Could we generalize the names of the parameters here? e.g. `links` -> `table_name`, etc.

Reply:
this method needs to take measures to prevent sql injection. please see [this reference from OWASP](https://cheatsheetseries.owasp.org/cheatsheets/SQL_Injection_Prevention_Cheat_Sheet.html) for information on some approaches.